pyproject.toml

```diff
@@ -139,6 +139,14 @@ write_to = "astropy/_version.py"
         name = "astropy.utils"
         path = "utils"
 
+    [[tool.towncrier.section]]
+        name = "astropy.utils.iers"
+        path = "utils.iers"
+
+    [[tool.towncrier.section]]
+        name = "astropy.utils.masked"
+        path = "utils.masked"
```

Comment:
Hmm... Maybe the bot is checking pyproject.toml from upstream. If you really want `utils.masked`, I can try to commit this part on `main` first and then you can rebase. Or you can move the change log under `utils` as before. Please let me know what you prefer. I apologize for any inconvenience caused.

https://github.com/pllim/actions-towncrier-changelog/blob/55409754c88f9743fcfc5113decbbcb6d98dedfd/check_changelog.py#L31

Reply:
Hmm, I think in analogy with `io.ascii`, etc., we should really have `utils.masked` and `utils.iers`, since there are corresponding labels. So, maybe go for adding those first? My only worry is that perhaps a difference is that plain `utils` exists, while plain `io` does not.